utils/client/config/config.go

```diff
@@ -44,12 +46,16 @@ const (
 
 	// EgressSelectorConfigFlagName is the name of the egress-selector-config flag.
 	EgressSelectorConfigFlagName = "egress-selector-config"
-)
 
-var (
-	log = ctrl.Log.WithName("client").WithName("config")
+	// QPS is the default value for allowed queries per second for a client.
+	QPS float32 = 20.0
```

Comment:
We might want to move the defaults here and remove it from the `common` package.

Reply:
you mean remove from `broker/common/defaults`? I will change this to ref the ones from `utils/client/config`

Reply:
addressed in latest commit